pallets/moonbeam-xcm-benchmarks/src/generic/benchmarking.rs

```diff
@@ -41,8 +41,55 @@ benchmarks! {
 
 	} : {
 		executor.bench_process(xcm)?;
-	} verify {
+	}
+
+	exchange_asset {
+	} : {
+		Err(BenchmarkError::Override(
+			BenchmarkResult::from_weight(T::BlockWeights::get().max_block)
```

Comment:
Unfortunately, I did not see this PR (I'm too overflowing), but using the max block weights is wrong (and 
theoretically not enough), we should use `Weight::MAX` instead

Reply:
Oh I noticed [substrate](https://github.com/paritytech/substrate/blob/master/frame/elections-phragmen/src/benchmarking.rs#L335) does the same so figured it should suffice. Could you please elaborate why the max block weight isn't enough in this context?

Reply:
Because theorically using the max block weight not disable the instruction, the instruction can fit in a block.

Reply:
I see, makes sense. I'll create another PR to change it

Reply:
Addressed in https://github.com/PureStake/moonbeam/pull/2275